SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -414,10 +440,28 @@ void USpatialSender::SendRPC(TSharedRef<FPendingRPCParams> Params)
 		// Target object was destroyed before the RPC could be (re)sent
 		return;
 	}
-
 	UObject* TargetObject = Params->TargetObject.Get();
+
+	if (USpatialActorChannel* Channel = NetDriver->GetOrCreateSpatialActorChannel(TargetObject, NetDriver->GetSpatialOSNetConnection()))
+	{
+		if (Channel->bCreatingNewEntity)
+		{
+			if (PackageMap->GetUnrealObjectRefFromObject(TargetObject) != FUnrealObjectRef::UNRESOLVED_OBJECT_REF)
```

Comment:
Can you make a comment as to when the TargetObject can be unresolved?

Reply:
My impression is that it should never happen. @improbable-valentyn do you remember if that could happen with singletons in multiworker?

Reply:
In the singleton case, `bCreatingNewEntity` shouldn't be true, so this should never happen, except if we failed to assign an entity ID from the pool (which means things will be very broken)

Reply:
Changed if to check.